tensorflow/java/src/gen/perl/tftypes-runall.pl

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/perl
```

Comment:
I think if you're actually changing the types, it's very handy to be able to generate one file at a time as you debug it. That functionality could be supported but at the cost of making the script more complex; I think it's useful to keep tftypes.pl as simple as possible.